app/jobs/invoices/abacus/create_yearly_invoices_job.rb

```diff
@@ -171,7 +171,7 @@ def update_progress(people_count)
   end
 
   def load_people(ids)
-    context.people_with_membership_years.where(id: ids).order(:id).includes(:roles)
+    context.people_with_membership_years.where(id: ids).order(:id)
```

Comment:
Macht das jetzt nicht n+1 queries? Braucht es hier nicht ein preload roles?

Reply:
Nein, das `context.people_with_membership_years` preloadet die `roles` bereits, und mit dem korrekten scope während das `includes(:roles)` den default scope verwendet der in diesem Kontext falsch ist.